gammapy/modeling/models/core.py

```diff
@@ -372,6 +373,35 @@ def to_dict(self, full_output=False):
         else:
             return {"components": models_data}
 
+    def to_parameters_table(self):
+        """Convert Models parameters to an astropy Table."""
+        pars = self.parameters.to_table()
+        rows = []
+        #Warning: splitting of parameters will break is source name has a "." in its name.
+        for name in self.parameters_unique_names:
```

Comment:
By now the `Parameter` object actually has a `.type` attribute which returns `spectral`, `spatial` or `temporal`. So you could actually modify `Parameters.to_table()` to include this information (not `Parameter.to_dict()`, because we use it for serialisation as well...). Then in the `.to_parameters_table()` you could just add the model name information...

Reply:
Done. 
`to_table()` now includes the model type and `to_parameters_table()` is much shorter and simpler to read.